network/wsPeer.go

```diff
@@ -530,7 +531,8 @@ func (wp *wsPeer) readLoop() {
 			wp.net.log.Warnf("wsPeer readLoop: received a TS response for a stale request from %s. %d bytes discarded", wp.conn.RemoteAddr().String(), n)
 			continue
 		}
-		slurper.Reset()
+
+		slurper.Reset(uint64(msg.Tag.MaxMessageSize()))
```

Comment:
Looks like outside of tests, the slurper is only reset to this one specific size (though maybe I missed something). If that's the case, why not just allow the slurper to set this value at construction and then not reload it?

Reply:
This is a for loop that runs for the lifetime of the peer. Each iteration of the for loop will have a different msg and therefore a different Tag and it's `MaxMessageSize()`